src/modules/pdt/doc/pdt_admin.xml

```diff
@@ -501,11 +501,8 @@ pd_translate("*", "$var(a)");
 		<para>
 		RPC Command Example:
 		</para>
-		<programlisting  format="linespecific">
-...
-# &kamcmd; _sdomain_ _prefix_ _domain_
-&kamcmd; kamailio.org 123
-...
```

Comment:
The dots here were on purpose to enclose the example within them in order to be more visible in README files. Also, the spaces/tabs added in front will make it show shifted a lot in the README, that's why most of the <programlisting  format="linespecific"> content is starting at the beginning of line and doesn't keep the indentation.

It can be merged, but then dots and the start at the beginning should be restored with a follow up commit.